src/components/PostList/PostList.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import './PostList.scss';
+
+import { CommentList } from '../CommentList';
+import { PostInfo } from '../PostInfo';
+import { UserInfo } from '../UserInfo';
+
+import { Post } from '../Types/Types';
+
+type Props = {
+  commonArr: Post[],
+};
+
+export const PostList: React.FC<Props> = ({ commonArr }) => (
+  <div>
+    {[...commonArr].map(post => {
```

Comment:
Wht you use [...comminArr] and what is it?

Reply:
just in case)